client/webpack.config.js

```diff
@@ -32,19 +31,29 @@ const config = {
       lib: path.resolve('./app/lib'),
       api: path.resolve('./app/api'),
       course: path.resolve('./app/bundles/course'),
+      testUtils: path.resolve('./app/__test__/utils'),
     },
   },
 
   optimization: {
     splitChunks: {
       chunks: 'all',
+      name: (_module, chunks, cacheGroupKey) => {
```

Comment:
This gives us the `[name]-[chunkhash].js` effect that we want (if I remember correctly).